content/learn/contribute/introduction.md

```diff
@@ -5,9 +5,9 @@ insert_anchor_links = "right"
 weight = 1
 +++
 
-Hey, so you're interested in contributing to Bevy! Feel free to pitch in on whatever interests you and we'll be happy to help you contribute.
+Hey, so you're interested in contributing to a Bevy project! Feel free to pitch in on whatever interests you and we'll be happy to help you contribute.
```

Comment:
```suggestion
Hey, so you're interested in contributing to Bevy! Feel free to pitch in on whatever interests you and we'll be happy to help you contribute.
```

(Non-blocking) I feel like "Bevy" encompasses all of its subprojects. Saying "a Bevy project" makes it sound like you could be contributing to a 3rd party Bevy plugin, not 1st party code.